web/client/epics/geoProcessingTools.js

```diff
@@ -184,9 +184,8 @@ export const checkWPSAvailabilityGPTEpic = (action$, store) => action$
                                     return Rx.Observable.of(setInvalidLayer(layerId, source));
                                 }
                                 return Rx.Observable.of(
-                                    getFeatures(layerId, source),
-                                    checkingWPS(false));
-                            })
+                                    getFeatures(layerId, source));
+                            }).concat([checkingWPS(false)])
```

Comment:
concat accepts a stream, not an array of events.

.concat(Rx.Obserrvable.of(checkingWPS(false))) should work. Maybe this is the reason why it was not working?